Dockerfile

```diff
@@ -1,12 +1,21 @@
-FROM alpine:latest
-MAINTAINER MessageBird <support@messagebird.com>
+FROM golang:1.11.1-alpine3.8 as build-env
```

Comment:
This image is very specific in which Go version is selected, could it be set to a more relaxed constraint?

Reply:
sure